### ISSUE TITLE
Style character sheet

### ISSUE THREAD

**Issue by stanavdb** [the-metalworks/cosmere-rpg#44]
The current design for the character sheet is functional, but doesn't yet match the awesome designs by @MangoFVTT. 
The general structure is mostly there, but a lot of clean up on the CSS/HTML is needed.

# Designs
![01_SheetMockups_Details](https://github.com/user-attachments/assets/69cd8096-adfe-41c6-8d20-c8de4ee52a3f)
![02_SheetMockups_Actions](https://github.com/user-attachments/assets/a5e58ccd-04af-473b-8565-655e5c4ff5d2)
![03_SheetMockups_Equipment](https://github.com/user-attachments/assets/61f371a4-f781-432a-869e-b12286d973d3)
![04_SheetMockups_Goals](https://github.com/user-attachments/assets/97ecb21c-a8b8-4ef6-8dd6-7f5f516cdfb7)
![05_SheetMockups_Effects](https://github.com/user-attachments/assets/706fad08-8b3e-4340-84ea-f653b68c8c71)

**Comment by PilotKayer** [comment]
I'll take care of it

**Comment by stanavdb** [comment]
Fair warning, the current CSS is an absolute mess. (entirely on me)

**Comment by zithith** [comment]
Can we clarify this ticket to include theming and theme selection.
Leave the current style as base and just apply the stormlight bits as a theme (probably in a separate stylesheet so it can be ported out easily.
Think that is kinda implied in the code currently, but nice to add it to the description here

**Comment by stanavdb** [comment]
> Can we clarify this ticket to include theming and theme selection. Leave the current style as base and just apply the stormlight bits as a theme (probably in a separate stylesheet so it can be ported out easily. Think that is kinda implied in the code currently, but nice to add it to the description here

Added.

**Comment by stanavdb** [comment]
On second thought, I have created a separate issue (#71) for theming so that #47 doesn't have to be dependent on this issue. I have made both this and #47 blocked by #71.

@PilotKayer Seeing as you're currently assigned to this, do you want me to assign you to #71 as well?

**Comment by TheFlyingPig** [comment]
Regarding theming, we should consider adding the option of transparency to the theme selection.
